axiom/nr_distributed_trace.c

```diff
@@ -481,14 +481,23 @@ void nr_distributed_trace_set_app_id(nr_distributed_trace_t* dt,
 }
 
 void nr_distributed_trace_set_trace_id(nr_distributed_trace_t* dt,
```

Comment:
I wonder if this interface would be cleaner if the padding was a value passed into the function.  If it is `0` then no padding is done, otherwise it is padded to the value supplied.  As it stands it is baked into the function that only W3C padding is supported.  The caller could decide what type of padding is required and pass in the appropriate value.

Reply:
I think that solution is over-engineered for the current problem. We have no reason to believe that there will ever be padding besides 16 prepended 0s.